docfiles/docs.js

```diff
@@ -261,7 +261,7 @@ function renderSnippets() {
 
 function responsiveResize(){
     // Add Responsive Adjustments depending on screen size 
-    $( window ).resize(function() {
+    window.addEventListener('resize', evt => {
```

Comment:
Watch out, you are in JavaScript like so you cannot use arrow functions, they are not supported in IE11. In general, testing your changes in IE11 is going to be the ultimate test.

Reply:
I totally forgot about good old IE11

Reply:
Still popular in schools.